### PR TITLE
80 refactor cleaner parser

### PR DESCRIPTION
**Pull request by Kangaroo6942**

### CODE REVIEW

**Review by OscarUlb — APPROVED** [review]
Looks good I like the new names. Thank you!

File: src/MensaWidget/MWDataGettingWrapper.class.st
Comment (lchristus) [inline src/MensaWidget/MWDataGettingWrapper.class.st:1]: we have here a instance variable that is canteenDict which is not in the class documentation but also has no accessors so I assume it is not used and can be removed

Comment (lchristus) [inline src/MensaWidget/MWDataGettingWrapper.class.st:1]: in get meals on date the constructor uses a lot of vars we can think if we do a line break there (sorry this was before already so can't comment on the line)

**Review by lchristus — CHANGES_REQUESTED** [review]
I haven't had the chance yet to look at it in Squeak but looking at the changes in here it looks very nice. Just two things that you could have done on top.

File: src/MensaWidget/MWDataGettingWrapper.class.st
Comment (Kangaroo6942) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: But canteenDict is a varibale that is in the scope of the MWDataGettingWrapper class, is it not?

Comment (Kangaroo6942) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: same here

Comment (lchristus) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: what you mean by same as above I don't see them correlating

Comment (Kangaroo6942) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: In my version of the code, there is not... Also, this variable exists in the DataGettingWrapper which's method is not the scope of this ticket

Comment (Kangaroo6942) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: i referred to the scope of the ticket, again you described something that is in the DataGettingWrapper class

Comment (Kangaroo6942) [reply src/MensaWidget/MWDataGettingWrapper.class.st:1]: I mean, I can change it here, if that is desired

**Review by lchristus — APPROVED** [review]
I honestly did not realize that the changes i requested was only in files that are not your responsibility

**Comment by Kangaroo6942** [comment]
i think that would be part of Oscars ticket then

**Comment by lchristus** [comment]
> i think that would be part of @OscarUlb ticket then

Yes I requested the same changes at Oscars branch aswell

**Review by lchristus — APPROVED** [review]
this is so perfect

**Review by OscarUlb — APPROVED** [review]

**Pull request merged**